yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -782,6 +783,17 @@ func getFreeDiskSpace(path string) (uint64, error) {
 	return freeSpace, nil
 }
 
+func cleanupDockerScriptMetadata() {
```

Comment:
the `/tmp` dir where the file is stored would be on the host's filesystem, but this code will run inside the container (i.e. a different file system), so I don't think it would correctly delete that file?

Either way, I think since this is a docker-specific logic, it would be good to have this logic inside the `yb-voyager-docker` script if possible.

Reply:
Since it's just a small tmp file, maybe we don't really have to delete as well?

Reply:
I agree.